src/analyzer/protocol/ntlm/ntlm-analyzer.pac

```diff
@@ -138,21 +138,22 @@ refine connection NTLM_Conn += {
 
 	function proc_ntlm_challenge(val: NTLM_Challenge): bool
 		%{
-		if ( ! ntlm_challenge )
+        	if ( ! ntlm_challenge )
```

Comment:
Here and elsewhere in this file is some space to tabs conversion and spurious whitespace insertion. Mind reverting/aligning?

Reply:
Done